themes/light/templates/addShows_addExistingShow.mako

```diff
@@ -12,6 +12,8 @@ const startVue = () => {
         el: '#vue-wrap',
         data() {
             return {
+                header: '${header}',
+                title: '${title}',
```

Comment:
This might not work as expected.
https://github.com/zzzeek/mako/blob/a582879d89ac6e37b9daa8b3c820ebedddad2aea/mako/runtime.py#L215-L234
That will raise a name error if you try and cast `str( )` on it (which is what `${   }` basically does)
I think this'll work:
```
header: '${header or ''}',
title: '${title or ''}',
```

Reply:
Obviously it will raise the exception only when header/title are actually undefined.

Reply:
TL;DR

After checking this,
1. My solution doesn't work if it's the variable is undefined (that's because [`strict_undefined=True`](http://docs.makotemplates.org/en/latest/runtime.html#context-variables)).
2. `header` and `title` are set to the value `FixME` by default when constructing `PageTemplate`. They should always be available in the template context, **and if they aren't - that's a bug to be fixed,** and it will throw a Mako error, like so:
```
2018-04-15 15:30:21 DEBUG    Thread_13 :: [6d54662] File C:\Medusa\medusa\server\web\core\base.py, line 147, in render
2018-04-15 15:30:21 DEBUG    Thread_13 :: [6d54662] File C:\Medusa\ext\mako\template.py, line 471, in render_unicode
2018-04-15 15:30:21 DEBUG    Thread_13 :: [6d54662] File C:\Medusa\ext\mako\runtime.py, line 838, in _render
2018-04-15 15:30:21 DEBUG    Thread_13 :: [6d54662] File C:\Medusa\ext\mako\runtime.py, line 873, in _render_context
2018-04-15 15:30:21 DEBUG    Thread_13 :: [6d54662] File C:\Medusa\ext\mako\runtime.py, line 899, in _exec_template
2018-04-15 15:30:21 DEBUG    Thread_13 :: [6d54662] File C:/Medusa/themes/light/templates/layouts/main.mako, line 128, in render_body
2018-04-15 15:30:21 DEBUG    Thread_13 :: [6d54662] File C:/Medusa/themes/light/templates/addShows_addExistingShow.mako, line 5, in render_scripts
2018-04-15 15:30:21 INFO     Thread_13 :: [6d54662] NameError: 'heade' is not defined
```

Reply:
@sharkykh yep, all the main vars in mako templates are set before we load anything in. Most using `FixMe` so we can easily find unset vars.